### PR TITLE
Fix ManifestWorkReplicaset featureGate

### PR DESCRIPTION
**Pull request by serngawy**

### CODE REVIEW

**Comment by qiujian16** [comment]
I am fixing another featuregate here https://github.com/open-cluster-management-io/registration-operator/pull/338 I think you could later use the similar way

**Comment by serngawy** [comment]
> I am fixing another featuregate here #338 I think you could later use the similar way

okay, sounds like same path

File: deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml [hunk 1]
```diff
@@ -13,3 +13,8 @@ spec:
     featureGates:
     - feature: DefaultClusterSet
       mode: Enable
+  # Enable ManifestWorkReplicaSet for e2e
+  workConfiguration:
```
Comment (qiujian16) [inline deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml]: we should not by default enable it in the example

**Review by qiujian16 — COMMENTED** [review]
/approve

looks good in general, just a minor comment

File: deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml (hunk 1 above)
Comment (serngawy) [reply deploy/cluster-manager/config/samples/operator_open-cluster-management_clustermanagers.cr.yaml]: okay, removed.

**Pull request merged**